check_api/src/main/java/com/google/errorprone/hubspot/HubSpotUtils.java

```diff
@@ -139,13 +149,24 @@ public static void init(JavacTask task) {
     });
   }
 
-  private static boolean isErrorHandlingEnabled(ErrorProneFlags flags) {
+  private static Supplier<Pattern> getGeneratedPathsPattern() {
+    return visitorState -> Optional.ofNullable(visitorState.errorProneOptions().getFlags())
```

Comment:
If we have access to the filesystem, we could also use [`PathMatcher` here](https://docs.oracle.com/javase/7/docs/api/java/nio/file/FileSystem.html#getPathMatcher(java.lang.String)) and take a glob as input

Reply:
That's pretty neat 👍